server/settings/settings.go

```diff
@@ -25,9 +26,16 @@ func (s *Server) Get(ctx context.Context, q *SettingsQuery) (*Settings, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	overrides := make(map[string]*v1alpha1.ResourceOverride)
+	for k := range argoCDSettings.ResourceOverrides {
+		val := argoCDSettings.ResourceOverrides[k]
+		overrides[k] = &val
+	}
 	set := Settings{
-		URL:         argoCDSettings.URL,
-		AppLabelKey: argoCDSettings.GetAppInstanceLabelKey(),
+		URL:               argoCDSettings.URL,
+		AppLabelKey:       argoCDSettings.GetAppInstanceLabelKey(),
+		ResourceOverrides: overrides,
```

Comment:
When every I edit `settings.go` I have to write a whole load of boiler-plate related to marshaling to/from the CM. Do you need this?

Reply:
Ignore this, I see those changes below.